target_postgres/json_schema.py

```diff
@@ -101,12 +101,14 @@ def is_nullable(schema):
 
 
 def _helper_simplify(root_schema, child_schema):
+    ret_schema = {}
+
     if not child_schema:
-        return child_schema
+        pass
```

Comment:
Could the if just be removed here then? Does it really not affect thel ogic?

Reply:
Actually, I think that case covered up another bug. Schemas are totally allowed to be completely empty. Those reference objects types. I'll add a better check for that.